rsyslog/init.sls

```diff
@@ -19,7 +19,7 @@ rsyslog:
     - template: jinja
     - source: {{ rsyslog.custom_config_template }}
     - context:
-      config: {{ salt['pillar.get']('rsyslog', {}) }}
+      config: {{ rsyslog|json }}
```

Comment:
```suggestion
        config: {{ rsyslog|json }}
```

Reply:
@kjkeane Good catch and resolution.  The fix above is not your fault but related to another problem.  See:

* https://github.com/saltstack-formulas/nginx-formula/pull/218

Reply:
@myii This should be updated now.